### PR TITLE
Feature redux managing app state

### PR DESCRIPTION
**Pull request by cristianoliveira**
It is a BIG PR but I had to change a lot of things to implement this. Also it is not managing all the state of the app, only player state.

### CODE REVIEW

**Comment by marceloboeira** [comment]
@marvin52

**Comment by marceloboeira** [comment]
Tests are green 🎈

**Comment by cristianoliveira** [comment]
WIP

**Comment by marceloboeira** [comment]
@cristianoliveira or beer 🍻

File: src/libs/feedcast-client/index.js [hunk 1]
```diff
@@ -1,17 +1,11 @@
-import EventEmitter from 'events'
-
-class FeedcastApi extends EventEmitter {
-  constructor(){
-    super();
-    this.apiUrl = process.env.REACT_APP_API_HOST
-  }
-
+const API_URL = process.env.FEEDCAST_API_HOST;
```
Comment (marceloboeira) [inline src/libs/feedcast-client/index.js]: https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables

Comment (marceloboeira) [inline src/libs/feedcast-client/index.js]: messed up shit ahha

Comment (cristianoliveira) [inline src/libs/feedcast-client/index.js]: @marvin52 yeah! Certo acabei esquecendo, vi que tu voltou a utilizar `REACT_APP*` em um outro commit

@marceloboeira Vamos manter a nomenclatura default por enquanto, não vejo necessidade de criar uma custom

Comment (marceloboeira) [inline src/libs/feedcast-client/index.js]: I don't understand your language, sorry.

Comment (marvin52) [inline src/libs/feedcast-client/index.js]: @cristianoliveira eu ia utilizar `FEEDCAST_` como padrão mas aí iria ser necessária mudanças no dotenv do projeto e também no buildpack.

Comment (marvin52) [inline src/libs/feedcast-client/index.js]: expensive @marceloboeira sorry

**Comment by cristianoliveira** [comment]
I finished most of the work. I think you can review and we can continue the other refactors after the merge

**Pull request merged**